dali/operators/reader/loader/numpy_loader.h

```diff
@@ -54,6 +54,24 @@ class NumpyParseTarget{
   }
 };
 
+struct NumpyFileWrapper {
+  Tensor<CPUBackend> image;
```

Comment:
```suggestion
  Tensor<CPUBackend> data;
```
I don't think we can call it an image anymore.